src/output/file_name.rs

```diff
@@ -396,9 +396,34 @@ impl<'a, 'dir, C: Colours> FileName<'a, 'dir, C> {
                 #[cfg(target_os = "windows")]
                 let abs_path = abs_path.strip_prefix("\\\\?\\").unwrap_or(&abs_path);
 
-                bits.push(ANSIString::from(format!(
-                    "{HYPERLINK_START}file://{abs_path}{HYPERLINK_END}"
-                )));
+                let distro_name = std::env::var("WSL_DISTRO_NAME").ok();
+                let path = if let Some(distro_name) = distro_name {
+                    if !abs_path.starts_with("/mnt/") {
```

Comment:
Thanks for the PR @opalmay! Could you have a look at the clippy warning on the failing CI check and fix it up? Ideally also run clippy locally to check there aren't any other warnings before amending and force-pushing the branch to get the CI to try again?

In case it's not clear, clippy is complaining that the code says "if not X then Y else Z" when it would be clearer as "if X then Z else Y".

Reply:
tests run through now and clippy seems to be happy as well ... so this should be resolved